src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -127,6 +137,13 @@ export const NavBar: React.FC<NavBarPropsType> = ({
     push: openUrlUsingRouter,
   } = useHistory();
 
+  const { isShown: isDropdownShown, toggle: toggleDropdownShown } = useShowHide(
+    false
+  );
+  const { isShown: isSearchShown, toggle: toggleSearchShown } = useShowHide();
+
+  const [isFocus, setIsFocus] = useState(false);
```

Comment:
`isFocus` what? 

I suggest update like the other variables 
(`isDropdown`...something or `isSearch`... something). 

It could be `isXyzFocused`, but `isInputFocused` is also too generic, thus might go further - `isXyzInputFocused`

Reply:
fixed in that PR https://github.com/sparkletown/sparkle/pull/2387